addons/binding/org.openhab.binding.bigassfan/ESH-INF/thing/thing-types.xml

```diff
@@ -12,12 +12,15 @@
 			<!-- Channels related to fan functions -->
 			<channel id="fan-power" typeId="fan-power" />
 			<channel id="fan-speed" typeId="fan-speed" />
-			<channel id="fan-direction" typeId="fan-direction" />
+			<channel id="fan-direction" typeId="fan-direction"></channel>
 			<channel id="fan-auto" typeId="fan-auto" />
 			<channel id="fan-whoosh" typeId="fan-whoosh" />
 			<channel id="fan-smartmode" typeId="fan-smartmode" />
+			<channel id="fan-learn-speed-min" typeId="fan-learn-speed-min" />
+			<channel id="fan-learn-speed-max" typeId="fan-learn-speed-max" />
 			<channel id="fan-speed-min" typeId="fan-speed-min" />
-			<channel id="fan-speed-max" typeId="fan-speed-min" />
```

Comment:
It wouldn't be backward compatible if it had actually worked.  :-(  One of the downsides of having to reverse engineer the protocol...

To the best of my knowledge, there's only one other user of the binding, and he wasn't using this feature (because it didn't work).

I don't think we need to add a release note for this.